test/functional/postalcode/postalcode.js

```diff
@@ -0,0 +1,99 @@
+define([
+	"globalize",
+	"json!cldr-data/supplemental/likelySubtags.json",
+	"json!cldr-data/supplemental/postalCodeData.json",
+	"../../util",
+	"json!./dummy/kr.json",
+	"json!./dummy/us.json",
+
+	"globalize/postalcode"
+
+], function( Globalize, likelySubtags, postalcode, util, kr, us ) {
+
+	QUnit.module( ".validatePostalCode( value )", {
+		setup: function() {
+			Globalize.load( likelySubtags, {
+				main: {
+					ko: {}
+				}
+			});
+			Globalize.locale( "ko" );
```

Comment:
@mwmw7 @jzaefferer I need your input:

Considering we require the `region` argument as stated above, how about changing the API to allow the static method only `Globalize.validatePostalCode()` and `Globalize.postalCodeValidator`, but not `Globalize.prototype.validatePostalCode` and `Globalize.prototype.postalCodeValidator`? Any instance method requires initialization using a locale. But, the postalcode validation actually requires no locale data. Having postal code validation methods statically on Globalize avoids confusion.